src/pages/Host/HostAmenity/HostAmenity.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import axios from 'axios';
+import HostAmenityCom from './HostAmenityCom';
+
+function HostRadio() {
+  const [data, setData] = useState({
+    productList: [],
+  });
+
+  const getData = useCallback(async () => {
+    try {
+      const response = await axios.get('/data/HostImg.json');
+      console.log(response);
+      setData({ productList: response.data });
```

Comment:
- state 의 구조를 수정한다면 업데이트는 다음과 같이 이루어질 것 같습니다.
```suggestion
      setProductList(response.data);
```